Documentation/building/testing-with-corefx.md

```diff
@@ -3,134 +3,144 @@ Testing with CoreFX
 
 It may be valuable to use CoreFX tests to validate your changes to CoreCLR or mscorlib.
 
-## Building CoreFX against CoreCLR
+# Building CoreFX against CoreCLR
 **NOTE:** The `BUILDTOOLS_OVERRIDE_RUNTIME` property no longer works.
 
 To run CoreFX tests with an updated System.Private.Corelib.dll, [use these instructions](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits).
 
-To build CoreFX against the updated System.Private.Corelib.dll - we need to update instructions.
+To build CoreFX against the updated System.Private.Corelib.dll - _we need to update instructions_.
 
-**Replace runtime between build.[cmd|sh] and build-tests.[cmd|sh]**
+## Replace runtime between build.[cmd|sh] and build-tests.[cmd|sh]
```

Comment:
I would be useful to say something about when one should use these instructions vs. the instructions from https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits

(I always use the CoreFX "testing-with-private-coreclr-bits" for my own workflow.)